core/modules/file/file.pages.inc

```diff
@@ -495,7 +495,8 @@ function file_manage_form($form, &$form_state, File $file) {
 
   $form['filename'] = array(
     '#type' => 'textfield',
-    '#title' => t('File name'),
+    '#title' => t('Description'),
```

Comment:
```suggestion
    '#title' => t('File display title'),
```

😉

Reply:
This is handled in the other PR, and that one updates all the other places too. Think we should keep them separate.

Reply:
Then revert all these changes here?

Reply:
Description is better than what we have now because it's consistent with everything else in the UI :) I'd like either of these to be able to get in w/o the other incase there's a hangup.

Reply:
These changes should be removed. We just changed these strings in https://github.com/backdrop/backdrop-issues/issues/4178 (in fact in a PR by @jenlampton at https://github.com/backdrop/backdrop/pull/2961).